ui/apps/platform/src/Containers/Clusters/DelegateScanning/Components/DelegatedScanningSettings.tsx

```diff
@@ -48,6 +81,36 @@ function DelegatedScanningSettings({
                         </FlexItem>
                     </Flex>
                 </FormLabelGroup>
+                <Flex className="pf-u-mt-md pf-u-mb-lg">
+                    <FlexItem>
+                        <FormLabelGroup
+                            label="Select default cluster to delegate to"
+                            isRequired
+                            fieldId="selectedClusterId"
+                            touched={{}}
+                            errors={{}}
+                        >
+                            <Select
+                                className="cluster-select"
```

Comment:
The picture looks borderless like Network Graph breadcrumb link for clusters.
![network-graph](https://github.com/stackrox/stackrox/assets/11862657/a9b3d3c3-77e9-494e-b98d-c136a4abcdc8)

The appearance on main dashboard looks more like what I would expect in a form:
![dashboard](https://github.com/stackrox/stackrox/assets/11862657/be70bc97-600c-474a-b9ab-83faa194dfd5)

Yet both have `className="cluster-select"` prop. What is the intended appearance?

Reply:
The Dashboard version doesn't have `className="cluster-select"`. (Maybe mistook this in the options list? `<Divider key="cluster-select-option-divider"...`)

I think that class was originally used to style that select in Network Graph 1.0, but is vestigial in NG 2.0 currently.

The behavior should match NG 2.0 (one cluster at a time), but the style is more like the Dashboard version.

I will point out the proliferation of styles to the UX team.

Reply:
Ah, I forgot--`cluster-select`, `namespace-select`, and `deployment-select` classes are used as Cypress selectors, too.

Reply:
Thank you for investigating. Sorry for my confusion. It was more haste, less speed.

Good search to find class names in selectors for integration tests. Let’s follow up to replace with aria attributes (and add them if needed) so that they do not break with style changes or simplifications. Dave has set an excellent example of adding overlooked accessibility attributes while he writes test selectors.